helm/cluster-apps-operator/values.yaml

```diff
@@ -13,7 +13,7 @@ provider:
 
 cni:
   mask: 16
-  subnet: 10.1.0.0/16
+  subnet: 10.1.0.0
```

Comment:
Can we instead remove `mask:`?

Reply:
no because configmap produces values with both parameters
```
          calico:
            subnet: '{{ .Values.cni.subnet }}'
            cidr: '{{ .Values.cni.mask }}'
```